lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/update-name.yaml

```diff
@@ -0,0 +1,228 @@
+        targetState: FAILED_COI
+      alternate-doc-invalid-dl:
+        targetJourney: FAILED
+        targetState: FAILED_COI
+
+  # Journey States
+
+  # WITHOUT ADDRESS
+
+  RESET_IDENTITY_WITHOUT_ADDRESS:
+    response:
+      type: process
+      lambda: reset-session-identity
+    events:
+      next:
+        targetState: DCMAW_UPDATE_NAME_ONLY
+
+  DCMAW_UPDATE_NAME_ONLY:
+    response:
+      type: cri
+      criId: dcmaw
+    parent: CRI_STATE
+    events:
+      next:
+        targetState: POST_DCMAW_SUCCESS_PAGE_NAME_ONLY
+      not-found:
+        targetJourney: FAILED
+        targetState: FAILED_COI
+      access-denied:
+        targetJourney: FAILED
+        targetState: FAILED_COI
+      temporarily-unavailable:
+        targetJourney: TECHNICAL_ERROR
+        targetState: ERROR
+      fail-with-no-ci:
+        targetJourney: FAILED
+        targetState: FAILED_COI
+
+  POST_DCMAW_SUCCESS_PAGE_NAME_ONLY:
+    response:
+      type: page
+      pageId: page-dcmaw-success
+      context: coi-no-address
+    events:
+      next:
+        targetState: FRAUD_CHECK_UPDATE_NAME
+
+  FRAUD_CHECK_UPDATE_NAME:
+    response:
+      type: cri
+      criId: fraud
+    parent: CRI_STATE
+    events:
+      next:
+  # TODO: Integrate with COI Check Lambda
+  #     targetState: COI_CHECK_LAMBDA
+        targetState: EVALUATE_GPG45_SCORES_UPDATE_NAME
+      enhanced-verification:
+        targetJourney: FAILED
+        targetState: FAILED_COI
+
+  # WITH ADDRESS
+
+  RESET_IDENTITY_WITH_ADDRESS:
+    response:
+      type: process
+      lambda: reset-session-identity
+    events:
+      next:
+        targetState: DCMAW_UPDATE_NAME_ADDRESS
+
+  DCMAW_UPDATE_NAME_ADDRESS:
+    response:
+      type: cri
+      criId: dcmaw
+    parent: CRI_STATE
+    events:
+      next:
+        targetState: POST_DCMAW_SUCCESS_PAGE_NAME_ADDRESS
+      not-found:
+        targetJourney: FAILED
+        targetState: FAILED_COI
+      access-denied:
+        targetJourney: FAILED
+        targetState: FAILED_COI
+      temporarily-unavailable:
+        targetJourney: TECHNICAL_ERROR
+        targetState: ERROR
+      fail-with-no-ci:
+        targetJourney: FAILED
+        targetState: FAILED_COI
+
+  POST_DCMAW_SUCCESS_PAGE_NAME_ADDRESS:
+    response:
+      type: page
+      pageId: page-dcmaw-success
+      context: coi-address
+    events:
+      next:
+        targetState: ADDRESS_AND_FRAUD_UPDATE
+
+  ADDRESS_AND_FRAUD_UPDATE:
+    nestedJourney: ADDRESS_AND_FRAUD
+    exitEvents:
+      next:
+        # TODO: Integrate with COI Check Lambda
+  #      targetState: COI_CHECK_LAMBDA
+        targetState: EVALUATE_GPG45_SCORES_UPDATE_NAME
+      enhanced-verification:
+        targetJourney: FAILED
+        targetState: FAILED_COI
+
+  # SHARED STATES
+
+  EVALUATE_GPG45_SCORES_UPDATE_NAME:
+    response:
+      type: process
+      lambda: evaluate-gpg45-scores
+    events:
+      met:
+        targetState: IPV_SUCCESS_PAGE_UPDATE_ADDRESS
```

Comment:
We probably want the target state here to just be `STORE_IDENTITY_BEFORE_SUCCESS_UPDATE_NAME` - the `sessionCredentialsTableReads` flag has now been dropped and we should always be storing the identity. Means the flag can be removed from below too.